Makefile

```diff
@@ -0,0 +1,50 @@
+.PHONY: help docker-start phpunit-full phpunit-mysql phpunit-mysql-without-dama phpunit-postgresql phpunit-postgresql-without-dama phpunit-mongo cs-fixer psalm
+
+MYSQL_URL="mysql://root:root@127.0.0.1:3306/zenstruck_foundry?charset=utf8"
+POSTGRESQL_URL="postgresql://root:root@127.0.0.1:5432/zenstruck_foundry?charset=utf8"
+MONGO_URL="mongodb://mongo:mongo@127.0.0.1:27017/mongo?compressors=disabled&amp;gssapiServiceName=mongodb"
+SQLITE_URL="sqlite://./var/app.db"
+
+.DEFAULT_GOAL := help
+
+help:
+	@fgrep -h "###" $(MAKEFILE_LIST) | fgrep -v fgrep | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+docker-start: ### create and run database containers
+	docker-compose up -d --build
+	./docker/wait-for-containers.sh
```

Comment:
this was actually easier to do that in a shell script. make syntax is quite cumbersome